src/components/GeneralInfo/DataCollectorsCard/DataCollectorsCard.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable camelcase */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import LoadingCard from '../LoadingCard';
+import DateFormat from '@redhat-cloud-services/frontend-components/DateFormat';
+import { TableComposable, Thead, Tr, Th, Tbody, Td, TableVariant, ExpandableRowContent } from '@patternfly/react-table';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { registered } from '../../../Utilities/index';
+
+const getDefaultCollectors = (entity) =>
```

Comment:
Can we add this function to [selectors.js](https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/src/components/GeneralInfo/selectors/selectors.js) please?